src/Features/Core/Portable/Completion/CommonCompletionUtilities.cs

```diff
@@ -241,5 +241,31 @@ public static bool TryRemoveAttributeSuffix(ISymbol symbol, SyntaxContext contex
 
             return true;
         }
+
+        public static bool ShouldPutCaretBetweenParenthesis(ISymbol symbol)
```

Comment:
My understanding is this should always return `false`

Reply:
I think Gen and I have posted somewhere above that we change the caret location. And later see if this makes sense, and we can change this to false whenever we like later.

Reply:
Did we not decide previously that `(` is the character that _always_ places the caret inside parentheses, while `;` is the character that _always_ places it at the end?

Reply:
https://github.com/dotnet/roslyn/pull/48851#issuecomment-721336201
Is this our conclusion that we want to stick to the design meeting decision? (And wait feedback)

Reply:
> Did we not decide previously that ( is the character that always places the caret inside parentheses, while ; is the character that always places it at the end?

This is my understanding as well. Although I think this method might need a better name, e.g. `IsInvocableWithParameters`

Reply:
If we don't want to change the caret position, then this method is meaningless.
Gonna remove it.